### PR TITLE
Fix inconsistent styling tags

### PR DESCRIPTION
**Pull request by mwitek**

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -4,5 +4,5 @@
     "packages/*"
   ],
   "npmClient": "yarn",
-  "version": "1.1.0"
+  "version": "1.2.3"
```
Comment (codenamev) [inline lerna.json]: This and the next 3 files have inconsistent versioning.  Is that expected?

File: packages/draft-js-import-markdown/src/MarkdownParser.js [hunk 2]
```diff
@@ -31,6 +31,7 @@ noop.exec = noop;
 
 var defaults = {
   gfm: true,
+  headers: true,
```
Comment (codenamev) [inline packages/draft-js-import-markdown/src/MarkdownParser.js]: Maybe rename to `headings`?  `headers` is a little ambiguous in this context.

File: lerna.json (hunk 1 above)
Comment (mwitek) [reply lerna.json]: kind of expected... I didn't know what I was doing while using https://lernajs.io/ but I do now :)  I'll sync these up

File: packages/draft-js-import-markdown/src/MarkdownParser.js (hunk 2 above)
Comment (mwitek) [reply packages/draft-js-import-markdown/src/MarkdownParser.js]: Sure

File: packages/draft-js-export-html/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "draft-js-export-html",
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (codenamev) [inline packages/draft-js-export-html/package.json]: `1.2.1`?

Comment (codenamev) [reply packages/draft-js-export-html/package.json]: nvm, I think I see

Comment (mwitek) [reply packages/draft-js-export-html/package.json]: yeah we didn't actually update this but the repo had changes that weren't published for this package.  We could remove this change all together 🤷‍♂️

**Pull request merged**